templates/events-user-created.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Template for My Events.
```

Comment:
I guess this file was erroneously re-added, since this has now moved to `event-list.php`